dnf/dnf-main.c

```diff
@@ -227,7 +227,11 @@ context_new (void)
   dnf_context_set_check_disk_space (ctx, FALSE);
   dnf_context_set_check_transaction (ctx, TRUE);
   dnf_context_set_keep_cache (ctx, FALSE);
-  dnf_context_set_cache_age (ctx, 0);
+
+  /* Set maximum cache age in seconds.
+   * G_MAXUINT - never expire, otherwise choose a lower value between this value
```

Comment:
I could probably dig out why this works as expected... You've added an explanatory comment which is a good idea here, but it doesn't make it clearer for me. You say:
> G_MAXUINT - never expire, otherwise choose a lower value ...

But that's not clear... _"otherwise"_ should be preceded by a condition, here I'm not sure in what case it chooses that lower value... Or, in other words, this just doesn't say how the expiration is evaluated exactly.

Reply:
@lukash OK I try it explain better. What do you think about this text?
```
/* Sets a maximum cache age in seconds. It is an upper limit.
 * The lower value between this value and "metadata_expire" value from repo/global
 * configuration file is used.
 * The value G_MAXUINT has a special meaning. It means that the cache never expires
 * regardless of the settings in the configuration files. */
```

Reply:
This one's good, I get it now, thanks :+1: